File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] res = new int[2 * n];
        int i = 0;
        while(i < n) {
            res[i] = nums[i];
            res[i + n] = nums[i++]; 
        }
        return res; 
    }
}

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        l = 0

        res = []

        for r in range(k - 1, len(nums)):
            res.append(max(nums[l: r + 1]))
            l += 1


        return res